test/adapter/polymorphic_test.rb

```diff
@@ -9,9 +9,19 @@ class PolymorphicTest < ActiveSupport::TestCase
           @picture = @employee.pictures.new(id: 1, title: 'headshot-1.jpg')
           @picture.imageable = @employee
 
-          @attributes_serialization = serializable(@picture, serializer: PolymorphicBelongsToSerializer) # uses default adapter: attributes
-          @json_serialization = serializable(@picture, adapter: :json, serializer: PolymorphicBelongsToSerializer)
-          @json_api_serialization = serializable(@picture, adapter: :json_api, serializer: PolymorphicBelongsToSerializer)
```

Comment:
pulled out into a helper method, as serialization variations grew by 2 more for empty relationship testing